simphony_metadata.yml

```diff
@@ -0,0 +1,415 @@
+    CUBA.ELECTROSTATIC_MODEL:
+      default: CUBA.CONSTANT_ELECTROSTATIC_FIELD_MODEL
+      
+  GRAVITY_MODEL:
+    definition: a simple gravity model 
+    parent: CUBA.PHYSICS_EQUATION
+    models: [CUBA.MESOSCOPIC, CUBA.CONTINUUM]
+    physics_variables: [CUBA.ACCELERATION]
+    CUBA.ACCELERATION:
+      default: [0, 0, 0]
+
+  CONSTANT_ELECTROSTATIC_FIELD_MODEL:
+    definition: A constant elecrostatic field model 
+    parent: CUBA.PHYSICS_EQUATION
+    models: [CUBA.MESOSCOPIC, CUBA.CONTINUUM]
+    physics_variables: [CUBA.ELECTRIC_FIELD, CUBA.CHARGE]
+    CUBA.ELECTROSTATIC_FIELD:
+      default: [0, 0, 0]
+  # note:GRAVITY and CUBA.ELECROSTATIC_FIELD are defined in cuba.yml. 
+
+  ISOTHERMAL_MODEL:
+    definition: Isothermal heat transport model, no transport of heat is assumed
+    parent: CUBA.PHYSICS_EQUATION
+    models: [CUBA.CONTINUUM]
+    physics_variables: []
+
+  THERMAL_MODEL:
+    definition: Non Isothermal heat transport model
+    parent: CUBA.PHYSICS_EQUATION
+    models: [CUBA.CONTINUUM]
+    physics_variables: [CUBA.TEMPERATURE, CUBA.HEAT_CONDUCTIVITY]
+      
+  INCOMPRESSIBLE_FLUID_MODEL:
+    parent: CUBA.PHYSICS_EQUATION
+    models: [CUBA.CONTINUUM]
+    definition: Incompressible fluid model
+    physics_variables: [CUBA.VELOCITY, CUBA.POSITION, CUBA.DENSITY, CUBA.VISCOSITY]
+    
+  LAMINAR_FLOW_MODEL:
+    parent: CUBA.PHYSICS_EQUATION
+    definition: Laminar model
+    models: [CUBA.CONTINUUM]
+    
+  SINGLE_PHASE_MODEL:
+    parent: CUBA.PHYSICS_EQUATION
+    definition: a single phase fluid model
+    models: [CUBA.CONTINUUM]
+
+  FREE_SURFACE_MODEL:
+    definition: Free surface model
+    parent: CUBA.PHYSICS_EQUATION
+    models: [CUBA.CONTINUUM]
+    physics_variables: [CUBA.SURFACE_TENSION]
+    
+  MIXTURE_MODEL:
+    definition: mixture (drift flux) model
+    parent: CUBA.PHYSICS_EQUATION
+    models: [CUBA.CONTINUUM]
+
+  RHEOLOGY_MODEL:
+    definition: Rheology model of a CFD fluid
+    parent: CUBA.PHYSICS_EQUATION
+    models: [CUBA.CONTINUUM]
+    default: CUBA.NEWTONIAN_FLUID_MODEL
+
+  NEWTONIAN_FLUID_MODEL:
+    definition: Newtonian ﬂuid model assuming the viscous stresses are proportional to the rates of deformation
+    parent: CUBA.RHEOLOGY_MODEL
+    models: [CUBA.CONTINUUM]
+  
+  POWER_LAW_VISCOSITY_MODEL:
+    definition: Power Law model for a variable viscosity function that is limited by minimum and maximum values 
+    parent: CUBA.RHEOLOGY_MODEL
+    models: [CUBA.CONTINUUM]
+    CUBA.MAXIMUM_VISCOSITY: 
+      default: 1e-3
+    CUBA.MINIMUM_VISCOSITY:
+      default: 1E-5
+    CUBA.LINEAR_CONSTANT:
+      default: 1e-5
+    CUBA.POWER_LAW_INDEX:
+      default: 1
+      
+  HERSCHEL_BULKLEY_MODEL:
+    definition: Herschel-Bulkley model combines the effects of Bingham plastic and power-law behavior in a fluid
+    parent: CUBA.RHEOLOGY_MODEL
+    models: [CUBA.CONTINUUM]
+    CUBA.INITIAL_VISCOSITY: 
+      default: 1e-3
+    CUBA.RELAXATION_TIME: 
+      default: 1
```

Comment:
SimPhoNy uses, currently, by default the SI system (MKS...). 
units should be introduced later. 
many new CUBA will have to be defined now in cuba.yml...